app/views/courses/_create_new_course.html.erb

```diff
@@ -1,20 +1,35 @@
-<h3>Create New Course</h3>
+<% @title = "Create New Course" %>
+
+<h4>Create New Course</h4>
```

Comment:
This is inconsistent with the Import Course tab header now, so I would change the header of that tab as well